sec-keys/openpgp-keys-danielkiper/openpgp-keys-danielkiper-20230215.ebuild

```
@@ -0,0 +1,26 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="OpenPGP keys used by Daniel Kiper"
+# I can't find a home page for him, so I'm just gonna put GRUB's since that's
+# currently his baby.
+# - xxc3nsoredxx
+HOMEPAGE="https://www.gnu.org/software/grub/"
+SRC_URI="
+	https://openpgpkey.net-space.pl/.well-known/openpgpkey/net-space.pl/hu/9b7hhehhfsjjw6h7bfzdesyjrhqhwgaq?l=dkiper
+		-> ${P}.asc
+"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~loong ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86"
+
+S=${WORKDIR}
+
+src_install() {
+	local files=( ${A} )
+	insinto /usr/share/openpgp-keys
+	newins - dkiper.asc < <(cat "${files[@]/#/${DISTDIR}/}")
```

This seems overly complex for a single file.
```suggestion
	insinto /usr/share/openpgp-keys
	newins "${DISTDIR}/${P}.asc" dkiper.asc
```

---

Sure. I just copypasta'd what looked to be in most `sec-keys/*` ebuilds.

---

Oh, I see. I guess we can keep it then.

---

I'll change the `.asc` to `.gpg` though and update the ebuilds.